### ISSUE TITLE
UserID is required to be an integer

### ISSUE THREAD

**Issue by drusellers** [msarchet/Flipper#3]
Any plans to support GUIDs?

related to #2

**Comment by msarchet** [comment]
Yes. I plan on supporting a more generic implementation of this, sorry it took so long to respond, I had my notifications turned off on this on accident and I put things on hold while I changed jobs.

**Comment by drusellers** [comment]
No worries. :)
